### DISCUSSION TITLE
Custom error page get filename and line number from error

### DISCUSSION THREAD

**Discussion by tychovbh** [vercel/next.js#11501] [Help]
# Feature request

## Is your feature request related to a problem? Please describe.

I'm trying to catch all errors in my website and sent them to slack so i'm aware of all errors. I created an _error.js file and in getInitialProps i'm sending the error to my back-end. It looks like this:

pages/_errors.js
```js
Error.getInitialProps = async ({res, err, req}) => {
    axios.post('/api/notifications', {
        notification: 'Front-end error',
        url: req ? req.path : Router.asPath,
        message: err ? err.message : 'Something went wrong',
        status: err && err.statusCode ? err.statusCode : res.statusCode
    })
})
```
Now this works and is great, but most of the time I have no idea were the error comes from. I'm missing information like in which file and line number the error took place.

Isn't this possible to retrieve this from err?

## Describe the solution you'd like

I would like to have more information from the error available like:

```js
err.fileName
err.linenumber
```

**Comment by timneutkens** [comment]
This is generally something that error reporting tools like https://sentry.io and https://www.bugsnag.com handle. You have to fetch the sourcemaps for every source line and map them in production.
